sa/model.go

```diff
@@ -232,26 +215,22 @@ func registrationToModel(r *core.Registration) (interface{}, error) {
 	if r.Contact == nil {
 		r.Contact = &[]string{}
 	}
-	rm := regModelv1{
+	rm := regModel{
 		ID:        r.ID,
 		Key:       key,
 		KeySHA256: sha,
 		Contact:   *r.Contact,
 		Agreement: r.Agreement,
 		InitialIP: []byte(r.InitialIP.To16()),
 		CreatedAt: r.CreatedAt,
+		Status:    string(r.Status),
 	}
 
-	return &regModelv2{
-		regModelv1: rm,
-		Status:     string(r.Status),
-	}, nil
+	return &rm, nil
 }
 
 func modelToRegistration(ri interface{}) (core.Registration, error) {
-	var rm *regModelv1
-	r2 := ri.(*regModelv2)
-	rm = &r2.regModelv1
+	rm := ri.(*regModel)
```

Comment:
Now that we're consolidating the two models, you can make the function signature more specific and say it takes `ri *regModel`. Similarly, you can change `registrationToModel` to return a `*regModel`.